packages/headless/src/features/search-and-folding/search-and-folding-request.ts

```diff
@@ -14,6 +12,8 @@ type StateNeededByExecuteSearchAndFolding = ConfigurationSection &
 
 export const buildSearchAndFoldingLoadCollectionRequest = async (
```

Comment:
Here also I don't think it needs to be async anymore (which implies you can remove awaits where this function gets called).